formatters.go

```diff
@@ -19,3 +21,24 @@ func PlainTextFormatter(decisions []*models.Decision) string {
 	sort.Strings(ips)
 	return strings.Join(ips, "\n")
 }
+
+func MicroTikFormatter(decisions []*models.Decision) string {
+	ips := make([]string, len(decisions))
+	for i, decision := range decisions {
+		var ipType string = "/ip"
+		if strings.Contains(*decision.Value, ":") {
+			ipType = "/ipv6"
+		}
+		ips[i] = fmt.Sprintf(
+			"%s firewall address-list add list=CrowdSec address=%s comment=\"%s for %s\"",
```

Comment:
@LaurenceJJones this name should be customizable or it will force users to have a list named CrowdSec in their router and it might make some users uncomfortable

Reply:
Okay, I understand the objective now. Should it be via the query string or via the configuration? or do you see a user case one user has multiple microtiks that want to use different list names from one endpoint?

Reply:
Upon further review I think we could make a more generic formatter that takes a golang template. We can merge both I just think if we are going to merge a function per format this is just too complicated. So allow a user to offer a custom template string would infinite possibilities as well as fix your issue here

Let me know how you feel about it we can do both

Reply:
> or do you see a user case one user has multiple microtiks that want to use different list names from one endpoint?

That's very possible indeed, a query param makes it easier so multiple users connect to the same host and do what they want with it

Reply:
> Upon further review I think we could make a more generic formatter that takes a golang template. We can merge both I just think if we are going to merge a function per format this is just too complicated. So allow a user to offer a custom template string would allow infinite possibilities as well as fix your issue here since they can edit the template

I think it's a good idea, but not very flexible with the query param in mind
but if we can combine replacing query param names with variables like a template then sure let's do it!
is it possible?